content_scripts/desktopCamera.js

```diff
@@ -453,25 +455,19 @@ createNameSpace('realityEditor.device.desktopCamera');
         // console.log(isExpanded);
     }
 
-    function logMessage(text) {
-        if (DEBUG_SHOW_LOGGER) {
-            closestObjectLog.innerText = text;
-        }
-    }
-
     // messageButtonIcon.src = '/addons/spatialCommunication/bw-message.svg';
 
     function panToggled() {
         if (threejsObject) {
             threejsObject.visible = knownInteractionStates.pan || knownInteractionStates.rotate || knownInteractionStates.scale;
         }
-        updateInteractionCursor(threejsObject.visible, '/addons/vuforia-spatial-remote-operator-addon/cameraPan.svg');
+        updateInteractionCursor(threejsObject.visible, '/addons/vuforia-spatial-remote-operator-cloud-edition/cameraPan.svg');
```

Comment:
does this change require the add-on to be renamed?

Reply:
It would! Thank you for bringing this up, I need to revert all these mentions of cloud-edition so that there's no renaming necessary